app/assets/stylesheets/rails_admin/base/theming.scss

```diff
@@ -12,12 +14,27 @@ body.rails_admin {
   /* room for upper navbar */
   padding-top: $navbar-height;
 
-  /* Application name */
-  .navbar .brand {
-    small {
-      color:#9d261d;
-      font-weight:bold;
-      line-height: 1;
+  .navbar {
+    /* Application name */
+    .brand {
```

Comment:
@tagliala Could you change in this PR  the css selector `.brand` in `.navbar-brand` please? :smile:
These are stylesheets for the bootstrap2 markup.

If you prefer I could do a PR after the merge of this one.

Thanks!